lab_1/start.py

```diff
@@ -2,17 +2,38 @@
 Concordance implementation starter
 """
 
-from main import read_from_file
 import os
+import main
 
 if __name__ == '__main__':
     #  use data.txt file to test your program
     current_dir = os.path.dirname(os.path.abspath(__file__))
-    data = read_from_file(os.path.join(current_dir, 'data.txt'))
-    stop_words = []
+    data = main.read_from_file(os.path.join(current_dir, 'data.txt'))
+    stop_words = ['of', 'a']
 
     #  here goes your logic: calling methods from concordance.py
 
-    RESULT = None
+    tokens = main.tokenize(data)
+    print(f'TOKENIZE >> {tokens[:10]}')
+
+    tokens = main.remove_stop_words(tokens, stop_words)
+    print(f'REMOVE STOPWORDS >> {tokens[:10]}')
+
+    freqs = main.calculate_frequencies(tokens[:5000])
+    print(f'CALCULATE FREQUENCIES >> {freqs[tokens[2]]} TIMES')
+
+    concordance = main.get_concordance(tokens, 'people', 2, 3)
+    print(f'GET CONCORDANCE >> {concordance[:2]}')
+
+    adjacent_words = main.get_adjacent_words(tokens, 'people', 2, 3)
+    print(f'GET ADJACENT WORDS >> {adjacent_words[:2]}')
+
+    sorted_concordance = main.sort_concordance(tokens, 'people', 2, 3, False)
+    print(f'SORT CONCORDANCE >> {sorted_concordance[:2]}')
+
+    main.write_to_file('report.txt', sorted_concordance)
+    print('WRITE TO FILE >> SUCCESSFUL')
+
+    RESULT = sorted_concordance
     # DO NOT REMOVE NEXT LINE - KEEP IT INTENTIONALLY LAST
-    assert RESULT == [(), ()], 'Concordance not working'
+    assert RESULT, 'Concordance not working'

```

Comment:
after == you should put an expected result, for example, 
assert RESULT == [['man', 'is', 'happy', 'the', 'dog', 'is'], ['dog', 'is', 'happy', 'but', 'the', 'cat']]